File: script-lang/sumdata.pl
#!/bin/perl
# Sebastian

use strict;

my $original   = 0;
my $compressed = 0;

open DATA, '<', 'data.log';
while (<DATA>) {
	my @token = split /\s+/;
	
	$original   += $token[4];
	$compressed += $token[5];
}
close DATA;

printf("Original size %d kb.\n",   $original/1000);
printf("Compressed size %d kb.\n", $compressed/1000);

File: script-lang/sumdata.rb
#!/usr/bin/ruby

sum_original = 0
sum_compressed = 0

File.open(ARGV[0]) do |f|
  while line = f.gets
    original, compressed = line.split[4,5]

    sum_original += original.to_i
    sum_compressed += compressed.to_i
  end
end

puts "Original size #{sum_original} kb"
puts "Original size #{sum_compressed} kb"
